src/cli.ts

```diff
@@ -118,6 +118,10 @@ if (require.main === module) {
       '-e, --electron-version <value>',
       "electron version to package, without the 'v', see https://github.com/electron/electron/releases",
     )
+    .option(
+      '--wvvmp',
+      "use a Widevine-enabled version of Electron (use at your own risk, it's unofficial, provided by CastLabs)",
```

Comment:
Adding mention of the word "DRM", for searchability

```suggestion
      "use a Widevine-enabled version of Electron for DRM playback (use at your own risk, it's unofficial, provided by CastLabs)",
```